eclair-core/src/main/scala/fr/acinq/eclair/blockchain/fee/FeeProvider.scala

```diff
@@ -21,13 +21,14 @@ import fr.acinq.bitcoin.scalacompat.{Satoshi, SatoshiLong}
 import scala.concurrent.Future
 
 /**
+ * Used to fetch feerates.
  * Created by PM on 09/07/2017.
  */
 trait FeeProvider {
   def getFeerates: Future[FeeratesPerKB]
 }
 
-case object CannotRetrieveFeerates extends RuntimeException("cannot retrieve feerates, channels may be at risk: ensure bitcoind estimatesmartfee correctly returns feerates and restart eclair")
+case object CannotRetrieveFeerates extends RuntimeException("Cannot retrieve feerates. Channels may be at risk. Ensure bitcoind estimatesmartfee correctly returns feerates and restart eclair")
```

Comment:
I don't see a compelling reason to change that?

Reply:
I reverted it, but I don't understand why you wouldn't want the error message to have correct punctuation and capitalization.

Reply:
Because those exception messages usually get included in logs in the middle of sentences so shouldn't be capitalized otherwise the punctuation is then incorrect, and the current punctuation isn't blatantly incorrect?